src/main/java/com/example/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,25 @@
 package com.example.onlinebookstore.dto;
 
+import com.example.onlinebookstore.validation.CoverImage;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.validator.constraints.ISBN;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
```

Comment:
Most likely, you have some restriction on length of title and author in the database, so put according validation for length. Also you can set some meaningful minimum length, but it's for you to decide, maybe there can be some book with one symbol name, but not empty for sure.